src/align/LinearAlignment.cpp

```diff
@@ -157,7 +157,7 @@ std::string OptimalTranscript(const std::string& target, int j1, int j2, const s
     //
     // Base case
     //
-    if ((j2 - j1 <= 1) || (i2 - i1 <= 1)) {
```

Comment:
Yes. The algorithm says if one interval is of size 0 or 1. Ours was eq less size of 2